src/hooks/resolvers/articles.resolvers.js

```diff
@@ -45,24 +45,6 @@ const resolveUserAddons = () => async (context) => {
   }
   debug(`'resolveUserAddons' for user: '${context.params.user.uid}' for ${uids.length} articles...`);
 
-  const collectables = await context.app.service('collectable-items').find({
-    authenticated: context.params.authenticated,
-    user: context.params.user,
-    query: {
-      resolve: 'collection',
-      item_uids: uids,
-    },
-  });
-
-  const collectablesIndex = lodash.keyBy(collectables.data, 'itemId');
-
-  const mapper = (d) => {
-    const collectableItemGroup = collectablesIndex[d.uid];
-    if (collectableItemGroup) {
-      d.collections = collectableItemGroup.collections;
-    }
-    return d;
-  };
 
   if (Array.isArray(context.result)) {
     context.result = context.result.map(mapper);
```

Comment:
mapper is required here! the PR should be modified to take into account this error